lib/matplotlib/tests/test_category.py

```diff
@@ -4,56 +4,77 @@
 
 import pytest
 import numpy as np
+import six
 
+from matplotlib.axes import Axes
 import matplotlib.pyplot as plt
 import matplotlib.category as cat
+from distutils.version import LooseVersion
 
-import unittest
+
+def _to_str(value):
```

Comment:
Not a fan of having two versions of the exact same function. Can the _to_str inside the StrFormatter of  call be pulled out into a private class method of StrFormatter so that it can be called in the same say convert can be called?